libs/ktxreader/include/ktxreader/Ktx2Reader.h

```diff
@@ -142,7 +142,7 @@ class Ktx2Reader {
 
         protected:
             Async() noexcept = default;
-            ~Async() = default;
+            virtual ~Async() = default;
```

Comment:
virtual method should always be defined in a cpp.

Reply:
done